conf/messages.en

```diff
@@ -239,7 +239,14 @@ claimRepaymentType.error.required = Select what you want to reclaim
 
 evidenceSupportingDocs.title = Required supporting documents
 evidenceSupportingDocs.heading = Required supporting documents
-evidenceSupportingDocs.paragraph1 = These are the documents we must have to process your claim.
+evidenceSupportingDocs.paragraph1.single = The documents you must have to process your claim are:
+evidenceSupportingDocs.paragraph1.multiple-small = The documents you must have <b>for each entry</b> to process your claim are:
```

Comment:
I think on NIDAC we would have avoided this markup in messages file. But quite possibly in NDRC it is moot.

Reply:
I would tend to agree.  But (a) we've done it elsewhere and (b) it is possibly easier to translate (compared with the alternative of creating three messages and concatenating them in the view).